risc0/zkvm/sdk/rust/guest/src/env.rs

```diff
@@ -34,8 +34,8 @@ use crate::{align_up, memory_barrier, sha};
 
 #[cfg(not(target_os = "zkvm"))]
 // Bazel really wants to compile this file for the host too, so provide a stub.
-/// Stub version of risc0_zkvm_platform::rt::host_sendrecv, re-exported for
-/// easy access through the SDK.
+/// This is a stub version of risc0_zkvm_platform::rt::host_sendrecv,
```

Comment:
```suggestion
/// This is a stub version of `risc0_zkvm_platform::rt::host_sendrecv`,
```

Or perhaps make an actual rustdoc link here.